applications: asset_tracker_v2: Update documentation (#7912)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

.. _app_data_types:

+----------------+----------------------------+-----------------------------------------------+--------------------+

> | Data type      | Description                | Identifiers                                   | No Data list entry |

Reviewer: ```suggestion
| Data type      | Description                | Identifiers                                   | String identifier for NOD list |
```

+================+============================+===============================================+====================+

> | Location       | GNSS coordinates           | APP_DATA_GNSS                                 | "gnss"             |

Reviewer 2: ```suggestion
| Location       | GNSS coordinates           | APP_DATA_GNSS                                 |``gnss``             |
```

+----------------+----------------------------+-----------------------------------------------+--------------------+
| Environmental  | Temperature, humidity      | APP_DATA_ENVIRONMENTAL                        | NA                 |
+----------------+----------------------------+-----------------------------------------------+--------------------+
| Movement       | Acceleration               | APP_DATA_MOVEMENT                             | NA                 |
+----------------+----------------------------+-----------------------------------------------+--------------------+
| Modem          | LTE link data, device data | APP_DATA_MODEM_DYNAMIC, APP_DATA_MODEM_STATIC | NA                 |
+----------------+----------------------------+-----------------------------------------------+--------------------+
| Battery        | Voltage                    | APP_DATA_BATTERY                              | NA                 |
+----------------+----------------------------+-----------------------------------------------+--------------------+

> | Neighbor cells | Neighbor cell measurements | APP_DATA_NEIGHBOR_CELLS                       | "ncell"            |

Reviewer 2: ```suggestion
| Neighbor cells | Neighbor cell measurements | APP_DATA_NEIGHBOR_CELLS                       | ``ncell``            |
```

+--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+
| Accelerometer threshold        | Accelerometer threshold in m/s². Minimal absolute value in m/s² for accelerometer readings to be considered valid movement.          | 10 m/s²        |
+--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+

> | No Data List (NOD)             | A list of strings that references :ref:`data types <app_data_types>` that will not be sampled by the application.                    | No entries     |

Reviewer 2: ```suggestion
| No Data List (NOD)             | A list of strings that references :ref:`data types <app_data_types>`, which will not be sampled by the application.                 | No entries     |
```

> | No Data List (NOD)             | A list of strings that references :ref:`data types <app_data_types>` that will not be sampled by the application.                    | No entries     |

Reviewer: I dont think I fully understand this. If the application wont sample these sensors, then how does it get the data? interrupts/events?

|                                | Used to disable sampling from sensor sources.                                                                                        | (Request all)  |

> |                                | For instance, when GNSS should be disabled in favor of location based on neighbor cell measurements.                                 |                |

Reviewer 2: ```suggestion
|                                | For instance, disabling GNSS in favor of location based on neighbor cell measurements.                                          |                |
```

> |                                | For instance, when GNSS should be disabled in favor of location based on neighbor cell measurements.                                 |                |

Reviewer: This sentence is not clear to me. Did you intend to say that "When GNNS should be disabled in favor of location based on neighbor cell measurements, the the string 'GNSS' must be added to this list." ?

Author: Jepp.

> |                                | The supported string list entries can be found in in the :ref:`data types <app_data_types>` table.                                   |                |

Reviewer 2: ```suggestion
|                                | The supported string list entries can be found in the :ref:`data types <app_data_types>` table.                                   |                |
```

+--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+
| Accelerometer threshold        | Accelerometer threshold in m/s². Minimal absolute value in m/s² for accelerometer readings to be considered valid movement.          | 10 m/s²        |
+--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+
| No Data List (NOD)             | A list of strings that references :ref:`data types <app_data_types>`, which will not be sampled by the application.                  | No entries     |
|                                | Used to disable sampling from sensor sources.                                                                                        | (Request all)  |
|                                | For instance, when GNSS should be disabled in favor of location based on neighbor cell measurements,                                 |                |

> |                                | the string identifier ''GNSS'' must be added to this list.                                                                           |                |

Reviewer 2: ```suggestion
|                                | the string identifier ``GNSS`` must be added to this list.                                                                           |                |
```
